src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ExpressionBuilder.java

```diff
@@ -91,12 +96,17 @@ private static Expression<BuiltCondition> buildExpressionRecursive(
       }
       conditionExpressions.add(subExpressionCondition);
     }
+
+    boolean hasInFilterThisLevel = false;
+    boolean InFilterThisLevelWithEmptyArray = true;
```

Comment:
naming convention: should be `inFilterThisLevelWithEmptyArray` (start with lower-case)